coralproject/talk
# Bulk approve/reject actions for Moderators

## What does this PR do?
-Ability to Select All comments and make an action in User History drawer
## How do I test this PR?
-In the User History drawer check 'Select All' and click on approve or reject. 

![image](https://user-images.githubusercontent.com/29279815/31554468-0c5fd99e-b014-11e7-8123-aee34f209c9e.png)

![image](https://user-images.githubusercontent.com/29279815/31554479-17096dec-b014-11e7-9354-5483b803e83d.png)

Feature #1027
